File: src/main/java/no/nav/service/pensjon/regler/to/FerdigstillEtteroppgjorUforetrygdRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.List;

import no.nav.domain.pensjon.regler.PenPerson;

import no.nav.domain.pensjon.regler.beregning2011.BeregnetUtbetalingsperiode;

public class FerdigstillEtteroppgjorUforetrygdRequest extends ServiceRequest {

    private List<BeregnetUtbetalingsperiode> beregnetUtbetalingsperiodeListe = new ArrayList<BeregnetUtbetalingsperiode>();

    private int etteroppgjorsar;

    /**
     * Unik id for personen det gjelder.
     * Lagt til av hensyn til arbeid med logger og feilanalyse.
     */
    private PenPerson penPerson;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/FerdigstillEtteroppgjorUforetrygdRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.PenPerson
import no.nav.pensjon.regler.domain.beregning2011.BeregnetUtbetalingsperiode

class FerdigstillEtteroppgjorUforetrygdRequest : ServiceRequest() {
    var beregnetUtbetalingsperiodeListe: List<BeregnetUtbetalingsperiode> = ArrayList()
    var etteroppgjorsar = 0

    /**
     * Unik id for personen det gjelder.
     * Lagt til av hensyn til arbeid med logger og feilanalyse.
     */
    var penPerson: PenPerson? = null
}